packages/meditrak-server/src/tests/routes/exportSurveyResponses/exportSurveyResponses.test.js

```diff
@@ -121,58 +138,83 @@ describe('exportSurveys(): GET export/surveysResponses', () => {
       xlsx.utils.aoa_to_sheet.resetHistory();
     });
 
-    it('Sufficient permissions: Should allow exporting an existing survey if users have both Tupaia Admin Panel and survey permission group access to the country of that survey', async () => {
-      await prepareStubAndAuthenticate(app);
-      await app.get(
-        `export/surveyResponses?surveyCodes=${survey1.code}&countryCode=${vanuatuCountry.code}`,
-      );
-
-      expect(xlsx.utils.aoa_to_sheet).to.have.been.calledOnce;
-
-      const exportData = xlsx.utils.aoa_to_sheet.getCall(0).args[0];
-      expectAccessibleExportDataHeaderRow(exportData);
-    });
+    describe('Should allow exporting a survey if users have Tupaia Admin Panel and survey permission group access to the corresponding countries', () => {
+      it('one country', async () => {
```

Comment:
Sorry, this is my bad, my PR comment was wrong 🙏 

```suggestion
      it('one survey', async () => {
```

Reply:
my bad for not realizing it too 😅